### PR TITLE
Containers: Increase the debbug information for k3s 

### PR DESCRIPTION
**Pull request by ilausuch**
- Related ticket: https://progress.opensuse.org/issues/128354
- Verification run: http://openqa.suse.de/tests/11024155

### CODE REVIEW

File: lib/containers/k8s.pm [hunk 1]
```diff
@@ -72,6 +72,11 @@ sub install_k3s {
     script_retry("kubectl get serviceaccount default -o name", delay => 60, retry => 3);
     # Await k3s to be ready and the api is accessible
     script_retry("kubectl get pods --all-namespaces", delay => 60, retry => 3);
+    assert_script_run("systemctl is-active k3s");
```
Comment (grisu48) [inline lib/containers/k8s.pm:75]: If k3s is not active, the previous command can not succeed, so this one is not necessary IMHO.

File: lib/containers/k8s.pm [hunk 2]
```diff
@@ -72,6 +72,11 @@ sub install_k3s {
...
+    assert_script_run("kubectl api-resources");
```
Comment (grisu48) [inline lib/containers/k8s.pm:76]: If this is for debugging purposes, I would consider putting it into a `record_info`, similar to lines 80 and 81.

File: lib/containers/k8s.pm (hunk 1 above)
Comment (ilausuch) [reply lib/containers/k8s.pm:75]: I suspect that sometimes the k3s fall down, so I would like to keep this or move to later. what do you think?

Comment (ilausuch) [reply lib/containers/k8s.pm]: Ok, removed. We have the sytemctl status k3s in case of failure

File: lib/containers/k8s.pm (hunk 2 above)
Comment (ilausuch) [reply lib/containers/k8s.pm]: Done

**Review by jlausuch — APPROVED** [review]

**Review by mloviska — APPROVED** [review]

**Pull request merged**